Expecto.Tests/Main.fs

```diff
@@ -2,13 +2,42 @@ module Main
 
 open Expecto
 open Expecto.Logging
+open OpenTelemetry.Resources
+open OpenTelemetry
+open OpenTelemetry.Trace
+open System.Threading
+open System.Diagnostics
+
+let serviceName = "Expecto.Tests"
+
+let logger = Log.create serviceName
+
+
+let resourceBuilder =
+    ResourceBuilder
+        .CreateDefault()
+        .AddService(serviceName = serviceName)
+
 
-let logger = Log.create "Expecto.Tests"
 
 [<EntryPoint>]
 let main args =
+  let activitySource = new ActivitySource(serviceName)
+  use traceProvider =
+          Sdk
+              .CreateTracerProviderBuilder()
+              .AddSource(serviceName)
+              .SetResourceBuilder(resourceBuilder )
+              .AddOtlpExporter()
+              .Build()
+  let tracer = traceProvider.GetTracer(serviceName)
```

Comment:
This is adding the OpenTelemetry Exporter to the test project, mostly for demo purposes. Could create a separate project if desired.